k8s-daemonset/k8s/linkerd-grpc.yml

```diff
@@ -28,10 +28,9 @@ data:
       label: outgoing
       experimental: true
       dtab: |
-        /srv                      => /#/io.l5d.k8s/default/grpc;
-        /svc/helloworld.Hello     => /srv/hello;
-        /svc/helloworld.World     => /srv/world;
-        /srv/world                => /srv/world-v1;
+        /srv       => /#/io.l5d.k8s/default/grpc;
+        /svc       => /$/io.buoyant.http.domainToPathPfx/srv;
+        /srv/World => /srv/world-v1;
```

Comment:
This was actually my initial reaction to this as well, but could not justify it so I gave up. But I think the most consistent dentry with our http examples would be:

```
/svc/helloworld.World => /srv/world-v1;
```